packages/main/src/Option.ts

```diff
@@ -1,8 +1,16 @@
-import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
 import customElement from "@ui5/webcomponents-base/dist/decorators/customElement.js";
 import slot from "@ui5/webcomponents-base/dist/decorators/slot.js";
+import event from "@ui5/webcomponents-base/dist/decorators/event.js";
 import property from "@ui5/webcomponents-base/dist/decorators/property.js";
-import type { IOption } from "./Select.js";
+import ValueState from "@ui5/webcomponents-base/dist/types/ValueState.js";
+
+import StandardListItem from "./StandardListItem.js";
+import ListItemType from "./types/ListItemType.js";
+import { IButton } from "./Button.js";
+import type { ListItemAccessibilityAttributes as OptionAccessibilityAttributes } from "./ListItem.js";
+import HighlightTypes from "./types/HighlightTypes.js";
```

Comment:
This type name is inconsistent, should be `Highlight` or `HighlightType`. Let's discuss harmonizing all type names.

Reply:
Added this to the planned changes issue #8461